### PR TITLE
Wip - few updates

### PR DESCRIPTION
**Pull request by Quentinchampenois**
#### :tophat: What? Why?

Few updates on UI

#### :pushpin: Related Issues
- Related to https://github.com/armandfardeau/decidim/pull/12#issuecomment-620143349

#### :clipboard: Subtasks
- [x] Swap the alert css class for "Examinated" and "Classified"
- [x] Display the answer date below the title
- [x] Allows admin to edit signature date collection for "examinated", "debatted" and "examinated" initiative state
- [x] Add tests
- [x] Backport commit fixing github actions

### CODE REVIEW

File: decidim-initiatives/app/forms/decidim/initiatives/admin/initiative_answer_form.rb [hunk 1]
```diff
@@ -42,6 +42,17 @@ def answer_date_allowed?
 
           state_updatable?
         end
+
+        private
+
+        def check_state
+          case context.initiative.state
+          when "published", "debatted", "examinated", "classified"
```
Comment (armandfardeau) [inline decidim-initiatives/app/forms/decidim/initiatives/admin/initiative_answer_form.rb]: I may be better to Use the constant defined on Decidim::Initiatives class, then add published?

File: decidim-initiatives/spec/system/initiative_spec.rb [hunk 2]
```diff
@@ -55,7 +55,7 @@
       end
 
       it "displays date" do
-        expect(page).to have_content(I18n.l(base_initiative.answer_date, format: :short))
+        expect(page).to have_content(I18n.l(base_initiative.answer_date, format: "%d/%m/%Y"))
```
Comment (armandfardeau) [inline decidim-initiatives/spec/system/initiative_spec.rb]: Please use I18n format

**Review by armandfardeau — CHANGES_REQUESTED** [review]
Thanks for your contribution, may you correct a few things before merging?

File: decidim-initiatives/spec/system/initiative_spec.rb (hunk 2 above)
Comment (Quentinchampenois) [reply decidim-initiatives/spec/system/initiative_spec.rb]: hi, thanks for the review. I agree with you and update it

File: decidim-initiatives/app/forms/decidim/initiatives/admin/initiative_answer_form.rb (hunk 1 above)
Comment (Quentinchampenois) [reply decidim-initiatives/app/forms/decidim/initiatives/admin/initiative_answer_form.rb]: Hi, thanks for your suggestion, I update it

**Pull request closed without merging**